packages/cmrjs/index.js

```diff
@@ -50,6 +52,7 @@ async function getMetadata(cmrLink) {
  * @returns {Object} - Full metadata as a JS object
  */
 async function getFullMetadata(cmrLink) {
+  deprecate('@cumulus/cmrjs/getFullMetadata(cmrLink)', '1.22.1');
```

Comment:
Couldn't find anywhere this was being used. This would need to be moved as well if it is being used.